packages/components/Notification/src/NotificationTokens.ios.ts

```diff
@@ -105,8 +105,8 @@ export const defaultNotificationTokens: TokenSettings<NotificationTokens, Theme>
     fontSize: 15,
     fontWeight: '400',
     minHeight: 52,
-    padding: globalTokens.spacing.m,
-    paddingVertical: globalTokens.spacing.s,
+    padding: globalTokens.size120,
```

Comment:
This should map to "`globalTokens.size160`". Given there is platform styling applied already, I think it will be better to map it to right translations based on endpoint for these. For components where there is no platform specific overrides, it's safe to map them all to win32 default.

Reply:
I'll double check what we should do here- iOS Notification was built before the token work started so I think it might have been referencing the old win32 tokens

Reply:
I have updated it to new "ios" size ramp based on tshirt size mapping. 
m->  size160
s -> size120

Reply:
I double checked and for Notification specifically what you have now looks good - from this [PR](https://github.com/microsoft/fluentui-react-native/pull/2251/files) the padding should be 16 and padding vertical should be 12. The snapshots don't seem to match though, think they just need to be updated again

Reply:
Works , @lyzhan7, 
i have mapping of  padding to 16 and padding vertical to 12 now. 

Can you once verify after taking latest once i merge this PR. Thanks